core/src/main/java/org/infinispan/remoting/transport/jgroups/JGroupsTransport.java

```diff
@@ -538,10 +540,9 @@ private void registerMBeansIfNeeded(String clusterName) {
          // Normally this would be done by CacheManagerJmxRegistration but
          // the channel is not started when the cache manager starts but
          // when first cache starts, so it's safer to do it here.
-         globalStatsEnabled = configuration.statistics();
-         if (globalStatsEnabled) {
-            // TODO Use the overloaded variant when available: https://issues.jboss.org/browse/JGRP-2394
-            JmxConfigurator.registerChannel(channel, jmxRegistration.getMBeanServer(), jmxRegistration.getDomain(), clusterName, true);
+         if (configuration.jmx().enabled()) {
+            ObjectName namePrefix = new ObjectName(jmxRegistration.getDomain() + ":" + ObjectNameKeys.MANAGER + "=" + ObjectName.quote(configuration.cacheManagerName()));
```

Comment:
@wburns is the `if` enough for Quarkus to not balk on the `ObjectName` static initializer?